app/grandchallenge/reader_studies/urls.py

```diff
@@ -85,6 +86,11 @@
         ReaderStudyExampleGroundTruthCSV.as_view(),
         name="example-ground-truth-csv",
     ),
+    path(
+        "<slug:slug>/ground-truth/copy-answers/",
```

Comment:
Maybe?

```suggestion
        "<slug:slug>/ground-truth/answers/create/",
```

Reply:
Also doubting. It's not really creating answers... or maybe it is but based off other answers? Creating answers is something that the CSV route also does.

Having had a think about this I think your suggestion is best. The /csv/ or /answers/ sort points to the source. answers/copy/ might be applicable but doesn't quite fit the rest of the URL routes in the app.